src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -31,6 +31,7 @@ export const ShowPracticeQuestionsBody = styled(PopupBody)`
 // tslint:disable-next-line: variable-name
 export const SectionTitle = styled.div`
   ${h4Style}
+  flex-shrink: 0;
```

Comment:
I think that it might be better to use the `flex` shortcut everywhere. Like `flex: 0 0 auto;` instead `flex-shrink: 0;`.

Reply:
`flex: 0 0 auto;` is a equivalent of
```
flex-grow: 0;
flex-shrink: 0;
flex-basis: auto;
```
Is there any advantage of using these properties here?

Reply:
In this example it doesn't really matter, but since these parameters influence each other, I think it is good practice to use the `flex` shorthand.